File: frontend/js/coworking.js
document.addEventListener('DOMContentLoaded', async () => {
  const container = document.getElementById('coworkingDetails');
  const params = new URLSearchParams(window.location.search);
  const id = params.get('id');

  const coworking = await getCoworkingById(id);

  container.innerHTML = `
    <div class="details-card">
      <div class="details-image">
        <img src="${coworking.imageUrl || 'images/placeholder.jpg'}" alt="${coworking.name}" loading="lazy">
      </div>

      <div class="details-info">
        <h1>${coworking.name}</h1>
        <p class="muted">${coworking.metro} • ${coworking.district}</p>
        <p>${coworking.description}</p>

        <p><strong>Адрес:</strong> ${coworking.address}</p>
        <p><strong>Рейтинг:</strong> ⭐ ${coworking.rating}</p>
        <p><strong>Уровень шума:</strong> ${coworking.noiseLevel}</p>
        <p><strong>Тип места:</strong> ${coworking.type}</p>

        <div class="amenities">
          ${coworking.amenities.map(item => `<span>${item}</span>`).join('')}
        </div>

        <h2>${coworking.pricePerHour} ₽ / час</h2>

        <a class="button button-green" href="booking.html?id=${coworking.id}">
          Забронировать
        </a>
      </div>
    </div>
  `;
});

File: backend/Models/Coworking.cs
namespace backend.Models;

public class Coworking
{
    public int Id { get; set; }

    public string Name { get; set; } = string.Empty;

    public string District { get; set; } = string.Empty;

    public string Metro { get; set; } = string.Empty;

    public string Address { get; set; } = string.Empty;

    public string Description { get; set; } = string.Empty;

    public decimal PricePerHour { get; set; }

    public double Rating { get; set; }

    public string NoiseLevel { get; set; } = string.Empty;

    public string Type { get; set; } = string.Empty;

    public string ImageUrl { get; set; } = string.Empty;

    public List<string> Amenities { get; set; } = [];
}